react-components/src/architecture/base/concreteCommands/cad/SetGhostModeCommand.ts

```diff
@@ -0,0 +1,33 @@
+/*!
+ * Copyright 2024 Cognite AS
+ */
+
+import { RenderTargetCommand } from '../../commands/RenderTargetCommand';
+import { type TranslateKey } from '../../utilities/TranslateKey';
+
+export class SetGhostModeCommand extends RenderTargetCommand {
+  // ==================================================
+  // OVERRIDES
+  // ==================================================
+
+  public override get tooltip(): TranslateKey {
+    return { key: 'GHOST_MODE', fallback: 'Ghost mode' };
```

Comment:
Do we remember to put this into translation?

Reply:
Yes